_data/openssl/X509_V_ERR_CERT_NOT_YET_VALID.yml

```diff
@@ -0,0 +1,27 @@
+error-code: X509_V_ERR_CERT_NOT_YET_VALID
+slug: x509-v-err-cert-not-yet-valid
+tags: [ openssl, time ]
+openssl-docs: |
+  The certificate is not yet valid: the notBefore date is after the current time.
+gnutls-code: GNUTLS_CERT_NOT_ACTIVATED
+gnutls-docs: |
```

Comment:
If I understand correctly, all the *other_library-docs* won't be eventually needed here. Is that true?

Reply:
Yes, there will be just a reference to the other code (and maybe a relationship status, i.e. equals/subset/superset). The documentation of the errors will be in the files of other libraries.